proto/user-service.proto

```diff
@@ -31,10 +34,19 @@ message UserDataRequest {
     string username = 2;
 }
 
-message UserRequest {
-    string id = 1;
+
+message LoginResponse {
+    bool authenticated = 1;
```

Comment:
There is apparently a oneof "attribute" you could probably use here.

https://developers.google.com/protocol-buffers/docs/proto3#oneof